src/components/LandingPage.jsx

```diff
@@ -1,7 +1,18 @@
-// src/components/LandingPage.jsx
-import { Link } from 'react-router-dom'
+import { useNavigate, Link } from 'react-router-dom'
+import { useEffect } from 'react'
+import { useAuth } from './AuthProvider'
 
 export default function LandingPage() {
+
+    const { user } = useAuth();
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        if (user) {
+            navigate('/dashboard');
+        }
+    }, [user, navigate]);
+
```

Comment:
Makes landing page aware if there is a user that is already logged in (avoid repeat logins)